okta/validator.go

```diff
@@ -0,0 +1,60 @@
+package okta
+
+import (
+	"errors"
+	"strings"
+)
+
+func validateConfig(c *config) (*config, error) {
+	var err error
+
+	err = validateOktaDomain(c)
+	if err != nil {
+		return nil, err
+	}
+
+	err = validateApiToken(c)
+	if err != nil {
+		return nil, err
+	}
+
+	return c, nil
+}
+
+func validateOktaDomain(c *config) error {
+	if c.Okta.Client.OrgUrl == "" {
+		return errors.New("your Okta URL is missing. You can copy your domain from the Okta Developer Console. Follow these instructions to find it: https://bit.ly/finding-okta-domain")
```

Comment:
Okta URL or Okta Domain?

Reply:
yes!

it is OrgUrl in our config, but Okta Domain in our library config check document

Reply:
Yeah, my concern is that both are used in your diff errors and can cause confusion.

Reply:
That message is from the wiki tech doc